libraries/AP_Scripting/examples/ahrs-set-origin.lua

```diff
@@ -0,0 +1,22 @@
+-- example script for using "set_origin()"
+-- sets the ekf origin if not already set
+
+
+function update ()
+
+    home = ahrs:get_home()
+    origin = ahrs:get_origin()
+    location = Location() location:lat(-353632640) location:lng(1491652352) location:alt(58409)
+
+    if origin then
+        gcs:send_text(0, "Unable to set EKF origin: already set")
+    return
```

Comment:
since this is just a return you could do

```
origin = assert(ahrs:get_origin(),"Unable to set EKF origin: already set")

```
on line 8

Reply:
Also with the early return here you don't need the `else` afterwards.
L13: indent of the return is wrong